libvips/foreign/exif.c

```diff
@@ -788,6 +788,20 @@ tag_is_ascii( ExifTag tag )
 		|| tag == EXIF_TAG_OFFSET_TIME
 		|| tag == EXIF_TAG_OFFSET_TIME_ORIGINAL
 		|| tag == EXIF_TAG_OFFSET_TIME_DIGITIZED
+		|| tag == EXIF_TAG_GPS_LATITUDE_REF
```

Comment:
I noticed that this tag overlaps with the IFD Interoperability index tag (`EXIF_TAG_INTEROPERABILITY_INDEX`), which is also an ASCII tag, but with limited choices (i.e. an enumeration) - see:
https://github.com/libexif/libexif/blob/4a7ab4cc1332afbc63c44a8a96c93421d6aa07a0/libexif/exif-tag.h#L175

I'm unsure if this could cause issues.

Reply:
I'm also unsure. Let's ignore this issue until someone complains.